fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -1362,15 +1363,27 @@ public LogicalPlan visitTableName(TableNameContext ctx) {
             scanParams = new TableScanParams(ctx.optScanParams().funcName.getText(), map);
         }
 
+        TableSnapshot tableSnapshot = null;
+        if (ctx.tableSnapshot() != null) {
+            if (ctx.tableSnapshot().TIME() != null) {
+                tableSnapshot = new TableSnapshot(stripQuotes(ctx.tableSnapshot().time.getText()));
+                tableSnapshot.setType(TableSnapshot.VersionType.TIME);
```

Comment:
The corresponding type has been set in the constructor of `TableSnapshot`, why should we set it again?

Reply:
Fixed. Thanks.